log/logger.go

```diff
@@ -1,12 +1,12 @@
 package log
 
-import cmlog "github.com/cometbft/cometbft/libs/log"
+import cmtlog "github.com/cometbft/cometbft/libs/log"
 
 // Logger is the interface that wraps the basic logging methods.
 type Logger interface {
 	Debug(msg string, keyvals ...interface{})
```

Comment:
@aaronc, do you mean having a CMTLogger interface here, inheriting a logger interface from core?